cloud-optimized-netcdf4-hdf5/index.qmd

```diff
@@ -140,7 +140,7 @@ For example, the following library defaults will impact performance and may be i
 * HDF5 library:
     * **Chunk cache:** The size of the HDF5's chunk cache by default is 1MB. This value is configurable. Chunks that don't fit into the chunk cache are discarded and must be re-read from the storage location each time. See also: [Improve HDF5 performance using caching](https://www.hdfgroup.org/2022/10/17/improve-hdf5-performance-using-caching/) and [h5py documentation: Chunk cache](https://docs.h5py.org/en/stable/high/file.html#chunk-cache).
     * **Page buffer size:** The [H5Pset_page_buffer_size](https://hdfgroup.github.io/hdf5/develop/group___f_a_p_l.html#title89) and [the page_buf_size argument to h5py.File](https://docs.h5py.org/en/stable/high/file.html) should match up with the page size to optimize reading the data in pages.
-* S3FS library: The S3FS library is a popular library for accessing data on AWS's cloud object storage S3. It has a default block size of 5MB ([S3FS API docs](https://s3fs.readthedocs.io/en/stable/api.html#s3fs.core.S3FileSystem)).
+* S3FS library: The S3FS library is a popular library for accessing data on AWS's cloud object storage S3. Note that S3FS has a default block size, which can be modified, see the [S3FS API docs](https://s3fs.readthedocs.io/en/stable/api.html#s3fs.core.S3FileSystem).
```

Comment:
How should we phrase this to be clear the default block size is subject to change, so you better check the docs to find out what it is.

Reply:
Please see https://github.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/153/commits/e3a284008b016876be904e2bf2ef463ff5123b71 and let me know if it is more clear.